core/src/avm2/globals/flash/net.as

```diff
@@ -20,13 +20,21 @@ package flash.net {
         this._aliasToClass[name] = object;
         this._classToAlias[object] = name;
     }
-    
-    public function getClassByAlias(name:String):Class {
-        if (this._aliasToClass[name]) {
-            return this._aliasToClass[name];
-        } else {
+
+    internal function _getClassByAlias(name:String):Class {
+        if (!this._aliasToClass.hasOwnProperty(name)) {
             return null;
         }
+
+        return this._aliasToClass[name];
+    }
+
+    public function getClassByAlias(name:String):Class {
+        var klass: Class = this._getAliasByClass(name);
```

Comment:
You probably meant `_getClassByAlias` here, not `_getAliasByClass`

Reply:
:sweat: thanks. I added a test.